pkg/tsdb/cloudwatch/log_alert.go

```diff
@@ -18,7 +18,7 @@ const (
 	alertPollPeriod  = time.Second
 )
 
-func (e *cloudWatchExecutor) executeLogAlertQuery(ctx context.Context, req *backend.QueryDataRequest) (*backend.QueryDataResponse, error) {
```

Comment:
Now that this is used for things beyond alerting should it go in a separate file, seems a bit weird to have it in a file called log_alert probably right? 

Also sort of basic on my part but how would one know if they should use a Sync vs Async Query?